### PR TITLE
BD : Fix streams + move JagoBD channels to playlist

### PR DESCRIPTION
**Pull request by BellezaEmporium**
- Fix Channel T1
- Move JagoBD links to specific playlists : https://www.jagobd.com/

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
I'll need to ask why some channels are Geo-Blocked. Channel I works on my side, said to be geo-locked.

**Comment by Dum4G** [comment]
Can confirm, Channel I is working for me

**Review by Sphinxroot — DISMISSED** [review]

**Review by freearhey — DISMISSED** [review]

File: channels/bd_jagobd.m3u [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="ATNBangla.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/ATN_Bangla.svg/207px-ATN_Bangla.svg.png" group-title="General",ATN Bangla (1080p) [Geo-blocked]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnbd-8-org.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ATNBanglaUK.bd" tvg-country="UK" tvg-language="Bengali" tvg-logo="http://atnbanglauk.tv/wp-content/uploads/2015/04/sized1.png" group-title="General",ATN Bangla UK (576p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/bd_jagobd.m3u:4]: is not geo-blocked for me

File: channels/bd_jagobd.m3u [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/boishakhitv-org.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BTVChittagong.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/CMUHtv3.png" group-title="Local",BTV Chittagong (720p) [Not 24/7]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvnational-ctg.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="BTVWorld.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/1Chamk9.png" group-title="General",BTV World (720p) [Not 24/7]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/btvbd-office-sg.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Channel9.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/f/ff/Channel9_bd.svg/350px-Channel9_bd.svg.png" group-title="Local",Channel 9 (720p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/bd_jagobd.m3u:18]: is not geo-blocked for me

File: channels/bd_jagobd.m3u [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel9hd.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Channel24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Channel24logo.svg/400px-Channel24logo.svg.png" group-title="News",Channel 24 (1080p)
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channel24-sg-e8e.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ChannelI.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://www.channelionline.com/wp-content/uploads/2019/07/channel-i-logo-1.png" group-title="General",Channel I (1080p) [Geo-blocked]
```
Comment (Dum4G) [inline channels/bd_jagobd.m3u:22]: is not geo-blocked for me

File: channels/bd_jagobd.m3u [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/channeli-8-org.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="ColosalTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/4kxxSCq.png" group-title="Entertainment",Colosal TV (720p) [Not 24/7]
+https://cdn.appv.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/colosal.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="DBCNews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://dbcnews.tv/img/dbc.png" group-title="News",DBC News (1080p) [Geo-blocked]
+https://live.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/dbcnews.stream/playlist.m3u8
```
Comment (Dum4G) [inline channels/bd_jagobd.m3u:27]: is not geo-blocked for me

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**